src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -432,6 +432,20 @@ public override int Order
         }
 
         /// <inheritdoc />
+        public override string Placeholder
+        {
+            get
+            {
+                if (DisplayMetadata.Placeholder == null)
+                {
+                    return null;
+                }
+
+                return DisplayMetadata.Placeholder();
```

Comment:
Would be nice to add a comment here that this shouldn't be cached because it's localizable.